src/config/delegates.json

```diff
@@ -1,4 +1,16 @@
-{
+i{
+ "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9": {
+    "name": "ReFiCol4🌎",
+    "address": "0xd42Bb7FE32cDf68045f49553c6f851fD2c58B6a9",
+    "logoUri": "/logos/validators/ReFiCol.jpg",
+    "date": "2024-05-01",
+    "links": {
+      "website": "https://reficolombia.org",
```

Comment:
This website doesn't work.

Reply:
We, don't have Website yet. Just the domain, but will put our Linktree in the meantime.